xdist/dsession.py

```diff
@@ -521,11 +94,13 @@ def pytest_collection(self):
 
     @pytest.mark.trylast
     def pytest_xdist_make_scheduler(self, config, log):
-        dist = config.getvalue("dist")
-        if dist == "load":
-            return LoadScheduling(config, log)
-        elif dist == "each":
-            return EachScheduling(config, log)
+        dist = config.getvalue('dist')
+        schedulers = {
+            'each': EachScheduling,
+            'load': LoadScheduling,
+            'loadsuite': LoadSuiteScheduling
+        }
+        return schedulers[dist](config, log)
```

Comment:
I wonder if we should make this experiment as a separate plugin instead? `pytest-xdist-loadsuite` for example. The addition of `pytest_xdist_make_scheduler` was done exactly to allow this kind of experimentation. 😁